SIG- 504 Created the scenario responsible to describe how to list Documents

jrodriguesmagal opened the pull request.

Commit:
Created the scenario responsible to describe how to list Documents

andreflourenco commented on dotnet/console/Console/Scenarios/ListDocumentsScenario.cs:
```diff
@@ -0,0 +1,40 @@
+                Order = PaginationOrders.Asc, //Order the list: Ascending = Oldest on top / Descending = Newest on top
+                OrganizationType = OrganizationType.Normal, //Define the organization type:  Normal, Personal , SharedWithMe
+                Limit = 2, //Define the number of Documents that you want to list 
+                IsConcluded = false, // List concluded documnets if it's "true"
```
@jrodriguesmagal  "documnets "

Commit:
Adapted the scenario to use the right parameters

Commit:
Adapted the scenario to use the right parameters
Adapted the scenario to use the right parameters
Adapted the scenario to use the right parameters

Commit:
Created ListDocument example
